Gemfile

```diff
@@ -36,6 +36,7 @@ gem 'bootsnap', '>= 1.1.0', require: false
 gem 'figaro'
 gem 'faraday'
 gem 'fast_jsonapi'
+gem 'date'
```

Comment:
Interesting that there is a gem for this rather than a built-in function

Reply:
I am actually not 100% sure. This may actually be built in.